sources/packages/backend/libs/sims-db/src/pagination-utils.ts

```diff
@@ -18,6 +18,19 @@ export function transformToApplicationEntitySortField(
     return orderByCondition;
   }
 
+  if (sortField === "dateSubmitted") {
```

Comment:
is it right to add this logic here?. This is a generic method used by all application-related pagination queries, which can have `dateSubmitted` and `status` for different entities. I think we need to review this

Reply:
Yes you are right @ann-aot `transformToApplicationEntitySortField` is for application related pagination queries. 

`status` in the method must be for application status and `submitted date`  here must be for application submitted date.

Also for this story we require multi level sort. status and submitted date. 

For the ticket I would recommend to create the build orderby condition method seperately at service level. 

We can talk about this if required.

Reply:
Changed as discussed. Thanks!